srcpkgs/pcc-libs/template

```diff
@@ -1,14 +1,21 @@
 # Template file for 'pcc-libs'
 pkgname=pcc-libs
-version=1.1.0
+_snapshot=20190420
+_distver=1.1.0
+version=${_distver}.${_snapshot}
 revision=1
-wrksrc=${pkgname}-${version}
+wrksrc=${pkgname}-${_distver}.BETA
 build_style=gnu-configure
 hostmakedepends="flex"
 short_desc="Portable C Compiler - Support Libraries"
-homepage="http://pcc.ludd.ltu.se/"
-license="BSD"
 maintainer="Juan RP <xtraeme@voidlinux.org>"
-distfiles="http://pcc.ludd.ltu.se/ftp/pub/pcc-releases/$pkgname-$version.tgz"
-checksum=dd4cb124da0b2edcda6ceb823c3ca8fc05919d35c89c34dec617517afe6bbfee
-archs="i686 x86_64"
+license="2-clause-BSD"
+homepage="http://pcc.ludd.ltu.se/"
+distfiles="http://pcc.ludd.ltu.se/ftp/pub/pcc-beta/${pkgname}-${_distver}.BETA-${_snapshot}.tgz"
```

Comment:
please fetch the license from somewhere with distfiles=

Reply:
The license is not available anywhere, couldn't find it.

Reply:
```
sed -n '4,29p' cc/driver/platform.c > LICENSE
vlicense LICENSE
```